src/library/trackset/crate/cratestorage.cpp

```diff
@@ -24,44 +20,41 @@ const QString CRATESUMMARY_TRACK_COUNT = "track_count";
 const QString CRATESUMMARY_TRACK_DURATION = "track_duration";
 
 const QString kCrateTracksJoin = QString(
-        "LEFT JOIN %3 ON %3.%4=%1.%2").arg(
-                CRATE_TABLE,
-                CRATETABLE_ID,
-                CRATE_TRACKS_TABLE,
-                CRATETRACKSTABLE_CRATEID);
-
-const QString kLibraryTracksJoin = kCrateTracksJoin + QString(
-        " LEFT JOIN %3 ON %3.%4=%1.%2").arg(
-                CRATE_TRACKS_TABLE,
-                CRATETRACKSTABLE_TRACKID,
-                LIBRARY_TABLE,
-                LIBRARYTABLE_ID);
+        "LEFT JOIN %3 ON %3.%4=%1.%2")
+                                         .arg(
+                                                 CRATE_TABLE,
+                                                 CRATETABLE_ID,
+                                                 CRATE_TRACKS_TABLE,
+                                                 CRATETRACKSTABLE_CRATEID);
```

Comment:
Is that how our pre-commit hooks formatted it? that looks strange.

Reply:
Yeah, I didn't check that, see https://github.com/mixxxdj/mixxx/pull/2715#discussion_r427163358 - fixing it rn